fabfile.py

```diff
@@ -1,29 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
It need to clear cache of npm, remove `node_modules`, rebuild node dependencies using `npm install`, build static site using `gulp build` and eventually restart apache server.

And because we removing `flask` we don't need to use virtualenv here so You can simplify context of command that will be executed.